styles/components/_modalWindow.scss

```diff
@@ -0,0 +1,159 @@
+// Positioning /
+// Display & Box Model /
+// Color /
+// Text /
+// Other /
+
+.modal {
+    // Positioning /
+    position: absolute;
+    left: 0px;
+    top: 0px;
+    z-index: 1;
+    // Display & Box Model /
+    width: 100vw;
+    height: 100vh;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    // Color /
+    background-color: rgba(0,0,0,.8);
+    &__container {
+        // Display & Box Model /
+        height: 889px;
+        width: 1200px;
```

Comment:
how about max-width of 1200px, what do you think?

Reply:
This was supposed to be just 1980x1020 version not responsive yet, but I guess it could stay like that for bigger screens also.